src/Component/Metadata/AsFilter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Metadata;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class AsFilter
+{
+    /**
+     * @param string $formType The form type class name to use for filter rendering
+     * @param string|null $type The filter type (defaults to FQCN of the class)
```

Comment:
`@param` which duplicates type hint without additional clarification for type like `class-string` shouldn't be added, in fact it will be removed with next CS fixes run. (Unless grid CS config outdated).

```suggestion
```

Reply:
I liked the documentation but let's remove it